wordpress-mobile/gutenberg-mobile
# Add image size picker improvements

Fixes #1452

### Companion PR's
Gutenberg PR: https://github.com/WordPress/gutenberg/pull/19232
WPiOS PR (for testing): https://github.com/wordpress-mobile/WordPress-iOS/pull/13052

### Changes

- Introduces cycle button UI for updating the size of an image from the BottomSheet menu in an Image Block.
- Removes duplicated default image size declarations and instead utilizes image sizes from the editor settings data store.

### Screenshots
![size-changes](https://user-images.githubusercontent.com/1103838/71149183-c8f18200-21fb-11ea-9b75-ad4d6f3e170e.gif)

See this comment and above comments for context: https://github.com/WordPress/gutenberg/pull/17245#pullrequestreview-301932626

### To Test:
1. Navigate to the Block Editor
2. Add an Image Block and upload an image
3. Click the gear for block settings
4. Tap Image Size row and verify value cycles through available sizes (Thumbnail, Medium, Large, Fullscreen).
5. Verify the Image size changes as you click through the menu.
6. Verify the Image Size is preserved if you update the post, leave, and return.

*Note this PR is mainly covering UI updates, for Image Size error handling with updating URL's from the network etc, see this issue: https://github.com/wordpress-mobile/gutenberg-mobile/issues/1593

Update release notes:
